### PR TITLE
Add GOL token

### PR DESCRIPTION
**Pull request by Gogolcoin**

### CODE REVIEW

**Comment by Gogolcoin** [comment]
Token Address: 0x083d41d6dd21ee938f0c055ca4fb12268df0efac
Token Name: GogolCoin
Token Decimals: 4
Token Symbol: GOL

Official homepage: https://gogolcoin.io/
Etherscan: https://etherscan.io/token/0x083d41d6dd21ee938f0c055ca4fb12268df0efac
Twitter: https://twitter.com/gogolcoin
Facebook: https://www.facebook.com/gogolcoin.io/
Telegram: https://t.me/joinchat/SvKR8Y9CwsvlrCSi
Discord: https://discord.com/invite/ETr7xUvf
Reddit: https://www.reddit.com/r/Gogolcoin/

**Review by erience — COMMENTED** [review]
looks good

**Review by erience — APPROVED** [review]

**Review by erience — APPROVED** [review]
Merge PR

**Comment by xrn** [comment]
Hi @Gogolcoin  - There is a new requirement that the logo has to be in SVG format. Can I ask you to add it as a replacement for the PNG one? Thanks

**Comment by Gogolcoin** [comment]
> Hi @Gogolcoin - There is a new requirement that the logo has to be in SVG format. Can I ask you to add it as a replacement for the PNG one? Thanks

Done, Thank you for the info

File: contract-map.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 {
+    "0x083D41d6DD21EE938f0c055CA4fb12268DF0EfaC": {
```
Comment (Ahmed-Majdoubi) [inline contract-map.json:2]: Looks fine.

**Review by Ahmed-Majdoubi — APPROVED** [review]

**Comment by xrn** [comment]
Can you please remove png file from this PR?

**Comment by Gogolcoin** [comment]
> Can you please remove png file from this PR?

File has been removed sucessfully.

**Review by TheMankou — APPROVED** [review]
Clean and tidy.

**Comment by Gogolcoin** [comment]
All errors solved, awaiting review.

Updates : 
- Probit IEO / Listing (Confirmed and in process.)
- Upcoming listing with Cointiger (After the ending of Probit's IEO)

**Comment by trastvolt** [comment]
Gogolcoin:master

**Review by MRabenda — APPROVED** [review]

**Comment by MRabenda** [comment]
@TheMankou can you solve conflicts? Seems that activity is fine now and we can merge this

**Comment by MRabenda** [comment]
@Gogolcoin can you solve conflicts? Seems that activity is fine now and we can merge this

**Pull request closed without merging**